users.md

```diff
@@ -1,6 +1,476 @@
 ---
 layout: contribute
-title: Bazel Users
+title: Who is using Bazel
 ---
 
-Find a community curated a list of Bazel users in the [Bazel GitHub wiki](https://github.com/bazelbuild/bazel/wiki/Bazel-Users)
\ No newline at end of file
+*Using Bazel? Send an email to [product@bazel.build](mailto:produc@bazel.build) to get added to the list.
+Feel free to also add your company [on StackShare](https://stackshare.io/bazel)*
+
+This page lists projects and companies that are known to use Bazel. This does not constitute an endorsement.
```

Comment:
This page lists some companies and open source projects that use Bazel.

(The order in the doc is companies and then projects, so ordering it that way in this sentence.)

Reply:
Fixed.